Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -0,0 +1,167 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MenuPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:commands="using:Microsoft.Toolkit.Uwp.SampleApp.Menu.Commands"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="MenuStyles.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+
+            <commands:NewProjectCommand x:Key="NewProject" />
+            <commands:NewFileCommand x:Key="NewFile" />
+            <commands:GenericCommand x:Key="GenericCommand" />
+        </ResourceDictionary>
+
+
+    </Page.Resources>
+
+    <Grid Background="{StaticResource Brush-Grey-05}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+
+        <controls:Menu HeaderButtonStyle="{StaticResource MenuHeaderButtonStyle}"
+                       MenuFlyoutStyle="{StaticResource MenuStyle}"
```

Comment:
Agree with @skendrot, if the styles are not necessary to get started, we should not have them in the sample app, it makes the control look much more complicated. If the styles are required, they should be part of the control.

I also see the need for having a way to showcase what is possible with the controls. Not sure if that should be in the sample app, or a sample outside of the app we can link to from the docs. We should probably have a larger discussion on this somewhere else :)